blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineNodeGraphVisitor.java

```diff
@@ -274,9 +274,13 @@ public void parallelStart(@Nonnull FlowNode parallelStartNode, @Nonnull FlowNode
                     status = new NodeRunStatus(genericStatus);
                 }
             }else{
-                times = new TimingInfo(TimingAction.getStartTime(branchStartNode)+System.currentTimeMillis(),
+                long startTime = System.currentTimeMillis();
```

Comment:
@abayer Doesn't look like right fix to me. This can be misleading, that is branchStartNode.getAction(TimingAction.class) will be null only in cases when this branch node has not started executing yet. Only case I see it working is if the assumption that if branchStartNode is available (branch endTime is not) that means its already started executing. Don't think this is true. This is situation where startBranchNode is created but it's timing info is not recorded yet, maybe it's waiting for executor to be available or just delay in writing to file system.

startTime of null or 0 feels like correct behavior, it tells client that we do not know it's start time vs giving current time without knowing its actually executing.